### ISSUE TITLE
Some troubles with Symfony Profiler

### ISSUE THREAD

**Issue by kefzce** [armetiz/LeezyPheanstalkBundle#76]
Hello sir, we got some problem, after migrating to this bundle from https://github.com/pheanstalk/pheanstalk

Symfony Profiler does not work correctly, and showing us this message:
`An error occurred while loading the web debug toolbar.`

And we got some messages in our log file below.

`[2018-12-08 20:25:08] request.CRITICAL: Uncaught PHP Exception UnexpectedValueException: "The profiler template "LeezyPheanstalkBundle:Profiler:pheanstalk.html.twig" for data collector "pheanstalk" does not exist." at /var/www/application/vendor/symfony/web-profiler-bundle/Profiler/TemplateManager.php line 90 {"exception":"[object] (UnexpectedValueException(code: 0): The profiler template \"LeezyPheanstalkBundle:Profiler:pheanstalk.html.twig\" for data collector \"pheanstalk\" does not exist. at /var/www/application/vendor/symfony/web-profiler-bundle/Profiler/TemplateManager.php:90)"} []`

We are using:

- Symfony 4.1 (https://github.com/armetiz/LeezyPheanstalkBundle/commit/33a71ed54bd8a7f19727c58a0ff26d91d89eb08f) Hope this compatibility works correctly ;)
- leezy/pheanstalk-bundle 3.3

Can u help us? many thanks.

**Comment by BenjaminLeibinger** [comment]
Hey!

under Symfony 4 the path to the resources has changed. The error can be fixed by the following config:

```
leezy_pheanstalk:
    profiler:
      template: '@LeezyPheanstalk/Profiler/pheanstalk.html.twig'
```

**Comment by msgagaev** [comment]
I'm sorry, but solution above don't fix this problem for Symfony 4.2.2

I solved the problem by replacing
{% extends 'WebProfilerBundle:Profiler:layout.html.twig' %} to {% extends '@WebProfiler/Profiler/layout.html.twig' %} in vendor/leezy/pheanstalk-bundle/src/Resources/views/Profiler/pheanstalk.html.twig

Maybe this problem may solved by another way. I don't know... 

What do you think about it?

**Comment by kefzce** [comment]
> I'm sorry, but solution above don't fix this problem for Symfony 4.2.2
> 
> I solved the problem by replacing
> {% extends 'WebProfilerBundle:Profiler:layout.html.twig' %} to {% extends '@WebProfiler/Profiler/layout.html.twig' %} in vendor/leezy/pheanstalk-bundle/src/Resources/views/Profiler/pheanstalk.html.twig
> 
> Maybe this problem may solved by another way. I don't know...
> 
> What do you think about it?

IDK,to be honest, I just fork  it and corrected on my side, and some fix on data collector too
